src/api/verify.ts

```diff
@@ -15,8 +15,25 @@ let lastToken: RadarTrackVerifiedResponse | null = null;
 let lastTokenNow: number = 0;
 let expectedCountryCode: string | null = null;
 let expectedStateCode: string | null = null;
+let lastIp: string | null = null;
 
 class VerifyAPI {
+  static async ipChanged() {
```

Comment:
Calls `GET /v1/ping`, returns `true` if IP changed

Reply:
nit: would call this function `checkIPChange`

Also, should this function only fire if `lastIp` is set? otherwise it will always trigger a change